Library/Tracking/KSRAnalytics.swift

```diff
@@ -598,6 +609,43 @@ public final class KSRAnalytics {
     )
   }
 
+  // MARK: - Block User Events
+
+  /**
+   Call when a user attempts or succeeds  to block a user.
```

Comment:
Nit: extra space and I'm a little unsure of what this comment means. Maybe "Call when a user attempts to block a user and when blocking succeeds" instead?